README.md

```diff
@@ -400,7 +400,7 @@ Options:
           separated list of accepted status codes. This example will accept 200, 201,
           202, 203, 204, 429, and 500 as valid status codes.
```

Comment:
It's unrelated to the changes in this pull request, but would you mind updating the help message here and in the Rust code to reflect the behavior of the code below? 🙏

Reply:
@mre I'm sorry, but I can't get which help message and code should be updated. For `-a, --accept <ACCEPT>` parameter, like clarify the default approach? Or something different?

Reply:
This part:

https://github.com/lycheeverse/lychee/blob/13f4339710d76831d9daf961584d796cee4847d2/lychee-bin/src/options.rs#L324-L326

It needs to be updated. It was incorrect before, but now is a good opportunity to fix it. 😉

Reply:
Coming back to this, I don't understand what I meant now either. I think I got confused, sorry. 😆